virttest/utils_windows/drive.py

```diff
@@ -37,3 +39,69 @@ def get_floppy_drives_letter(session):
     """
     cond = "MediaType!=0 AND MediaType!=11 AND MediaType!=12"
     return _logical_disks(session, cond=cond, props=["DeviceID"])
+
+
+def rescan_windows_disk(session):
+    """
+    Rescan disk in windows guest.
+
+    :param session: Session object.
+    """
+    rescan_cmd = "echo rescan > cmd"
+    rescan_cmd += " && echo exit >> cmd"
+    rescan_cmd += " && diskpart /s cmd"
+    logging.debug("Rescan disk")
+    session.cmd(rescan_cmd)
+
+
+def extend_shrink_windows_disk(session, did, disk_letter, operation=""):
```

Comment:
It is better to separate this into two functions, one for extending, the other for shrinking.

Reply:
Hi xuhan, I considered create two functions before, but if I separate to two functions, the code on 69 - 83 lines will need to write 2 times, both extend and shrink need these steps. In order to reduce the duplicated codes,  merged them to 1 function.